src/test_fixture/world.rs

```diff
@@ -122,4 +154,24 @@ where
             join_all(zip(contexts, input_shares).map(|(ctx, shares)| helper_fn(ctx, shares))).await;
         <[_; 3]>::try_from(output).unwrap()
     }
+
+    async fn malicious<'a, 'b, F, O, M, H, R>(&'a self, _input: I, mut _helper_fn: H) -> [O; 3]
+    where
+        'a: 'b,
+        A: IntoMalicious<F, M>,
+        F: Field,
+        O: Send + Debug,
+        M: Send,
+        H: FnMut(MaliciousContext<'b, F>, M) -> R + Send,
+        R: Future<Output = O> + Send,
+        Standard: Distribution<F>,
+    {
+        // self.semi_honest(input, |ctx, args| async {
+        //     let v = MaliciousValidator::new(ctx);
+        //     let m_share = args.upgrade(v.context()).await;
+        //     helper_fn(v.context(), m_share).await
```

Comment:
I think we should also call `v.validate()` after?

Reply:
Definitely.